src/skeleton/__tests__/skeleton-animation.scenario.js

```diff
@@ -19,6 +19,7 @@ export default function Scenario() {
           <div key={index}>
             <Skeleton
               animation={true}
+              rows={1}
```

Comment:
We spoke about how these scenarios should not have row props provided, and the override should be renamed to `Root`. Is that possible?